docker/Dockerfile

```diff
@@ -21,7 +21,7 @@ RUN mvn -B --no-transfer-progress package -DskipTests=true -pl autoconfigure -am
 RUN \
   cd /code && mkdir -p /zipkin-storage-kafka && cp autoconfigure/target/zipkin-autoconfigure-storage-kafka-*-module.jar /zipkin-storage-kafka && cd /zipkin-storage-kafka && jar xf *.jar && rm *.jar
 
-FROM openzipkin/zipkin:master
```

Comment:
this is temporary because I broke something? or is it intentional? not sure if this dockerfile is to reflect the master tag or the lastest

Reply:
Intentional, to align it with maven dependency version. For instance, could be that when no-autoconfig get merged we'll require to adapt source-code and config before we update to a new version (?)

Reply:
makes sense, was asking to clarify